glide-core/src/client/value_conversion.rs

```diff
@@ -182,6 +183,38 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
+        // command returns nil or an array of 2 elements, where the second element is a map represented by a 2D array
```

Comment:
not a map of map?  Is the first element a key but allows for duplicates?

Reply:
note: maybe point out that this method expect an array of exactly two elements?

Reply:
It is checked on line 198